### PR TITLE
Ruby: fix some more style-guide violations in the alert-messages

### PR DESCRIPTION
**Pull request by erik-krogh**
One last PR, where I went through the alerts produced by QL-for-QL.

Ignore the alerts. I'm also changing [JS](https://github.com/github/codeql/pull/10727) and [Python](https://github.com/github/codeql/pull/10729)

Drive-by fix of a deprecation warning that caused the CI to fail on `main`.

### CODE REVIEW

File: ruby/ql/lib/codeql/ruby/security/UnsafeDeserializationCustomizations.qll [hunk 1]
```diff
@@ -48,13 +48,12 @@ module UnsafeDeserialization {
...
-   * An argument in a call to `YAML.load`, considered a sink
-   * for unsafe deserialization. The `YAML` module is an alias of `Psych` in
-   * recent versions of Ruby.
+   * An argument in a call to `YAML.load`, considered a sink for unsafe
+   * deserialization.
    */
   class YamlLoadArgument extends Sink {
     YamlLoadArgument() {
-      this = API::getTopLevelMember(["YAML", "Psych"]).getAMethodCall("load").getArgument(0)
+      this = API::getTopLevelMember("YAML").getAMethodCall("load").getArgument(0)
```
Comment (aibaars) [inline ruby/ql/lib/codeql/ruby/security/UnsafeDeserializationCustomizations.qll:56]: Why did you remove `Psych`? Those two modules are the same thing in Ruby 3.0

File: ruby/ql/src/queries/security/cwe-912/HttpToFileAccess.ql [hunk 2]
```diff
@@ -18,4 +18,5 @@ import codeql.ruby.security.HttpToFileAccessQuery
 
 from Configuration cfg, DataFlow::PathNode source, DataFlow::PathNode sink
 where cfg.hasFlowPath(source, sink)
-select sink.getNode(), source, sink, "$@ flows to file system.", source.getNode(), "Untrusted data"
+select sink.getNode(), source, sink, "Write to file system depends on $@.", source.getNode(),
+  "Untrusted data"
```
Comment (aibaars) [inline ruby/ql/src/queries/security/cwe-912/HttpToFileAccess.ql:22]:
```suggestion
  "untrusted data"
```

**Review by aibaars — COMMENTED** [review]
Looks mostly good.

File: ruby/ql/lib/codeql/ruby/security/UnsafeDeserializationCustomizations.qll (hunk 1 above)
Comment (erik-krogh) [reply ruby/ql/lib/codeql/ruby/security/UnsafeDeserializationCustomizations.qll:56]: No idea why that happened. Maybe it was a bad merge?

**Review by aibaars — APPROVED** [review]

**Review by aibaars — APPROVED** [review]
